internal/api/approval/approval.go

```diff
@@ -293,13 +296,10 @@ func contentsUrlToRelDir(contentsUrl string) (string, error) {
 	return strings.Join(pathParts[3:len(pathParts)-1], "/"), nil
 }
 
-func (a *Approval) computeRulesForTargetBranch(ctx context.Context, pr *PR) ([]configuration.Rule, error) {
-	// Get the configuration for approvals in the current repository.
-	cfg, err := a.client.GetConfiguration(ctx, pr.OwnerLogin, pr.RepoName)
```

Comment:
pulled out of this function and used in the parent function instead as the configuration is needed there